docs/deploy/subnet_discovery.md

```diff
@@ -21,10 +21,12 @@ Private subnets are used for internal load balancers. These subnets must have th
 
 
 ## Common tag
-Both the public and private subnets must be tagged with the cluster name as follows:
+In version 2.1.1 and older, both the public and private subnets must be tagged with the cluster name as follows:
 
 | Key                                     | Value                 |
 | --------------------------------------- | --------------------- |
 | `kubernetes.io/cluster/${cluster-name}` | `owned` or `shared`   |
 
  `${cluster-name}` is the name of the kubernetes cluster
+ 
+ In version 2.1.2 and newer, this requirement is not [needed](https://github.com/kubernetes-sigs/aws-load-balancer-controller/pull/1773). 
```

Comment:
Lets reword it as -
"The cluster tag is not required in v2.1.2 and newer releases."

The PR need not be referenced, version should be sufficient.

Reply:
done and done